src/adapters/claude.ts

```
@@ -69,19 +69,39 @@ export class ClaudeAdapter extends IDEAdapter {
       const projectSlug = this.config.projectName.toLowerCase().replace(/\s+/g, '-');
 
-      files.push({
-        path: path.join(prpPath, `${projectSlug}-prp.md`),
-        content: await generatePRP(this.config, 'base'),
-        description: 'Base implementation PRP',
-      });
-
-      // Add planning PRP for complex projects
-      if (this.config.timeline === 'enterprise' || this.config.teamSize !== 'solo') {
+      // In retrofit mode, generate PRPs for planned features
+      if (this.config.isRetrofit && this.config.plannedFeatures && this.config.plannedFeatures.length > 0) {
```

ESLint

[error] Replace `this.config.isRetrofit·&&·this.config.plannedFeatures·&&·this.config.plannedFeatures.length·>·0` with `⏎········this.config.isRetrofit·&&⏎········this.config.plannedFeatures·&&⏎········this.config.plannedFeatures.length·>·0⏎······`

(prettier/prettier)